Makefile

```diff
@@ -24,7 +24,7 @@ clean-jar:
 
 # remove local dev db and jar
 .PHONY: clean-all
-clean-all: clean-sqlite clean-jar
+clean-all: clean-sqlite clean-psql clean-jar
```

Comment:
since psql is not being installed in default docker and in local environment, I would remove it from here. In the current state, it will error with "command not found: psql" which is as expected